frontend/src/properties/propertysections/RadioPropertySection.tsx

```diff
@@ -149,6 +155,7 @@ export const RadioPropertySection = ({ editingContextId, formId, widget, subscri
               control={<Radio color="primary" onFocus={onFocus} onBlur={onBlur} data-testid={option.label} />}
               label={option.label}
               key={option.id}
+              disabled={readOnly}
```

Comment:
The permission component has not been removed in this commit

Reply:
Done